wordcounter.rb

```diff
@@ -0,0 +1,30 @@
+class Counter
+  attr_accessor :dictionary, :filepath
+
+  def initialize(filepath)
+    @filepath = filepath
+    @dictionary = {}
+  end
+
+  def populate_dictionary
+    text_file = File.open(@filepath)
+    contents = text_file.read
```

Comment:
A file's content can be read into a string in one line with `contents = IO.read(@filepath)` http://ruby-doc.org/core-2.2.2/IO.html#method-c-read

That way you don't need to remember to call `text_file.close` after you're done using it. 😉 

Since `File` is a subclass of `IO`, I recommend using `contents = File.read(@filepath)` to make the intent a bit clearer.